src/libs/actions/Task.js

```diff
@@ -672,6 +672,25 @@ function isTaskAssigneeOrTaskOwner(taskReport, sessionAccountID) {
     return sessionAccountID === getTaskOwnerAccountID(taskReport) || sessionAccountID === getTaskAssigneeAccountID(taskReport);
 }
 
+/**
+ * Focus the task description text input and place the cursor at the end of the value (if there is a value in the input).
+ *
+ * @param {Object} input the input element
+ */
+function focusAndUpdateTaskDescriptionInputRange(input) {
```

Comment:
This is not specific to the task descriptions but to any input that exists and has multiple lines right?

Reply:
Let's rename this and/or move it to a more general util file

Reply:
> This is not specific to the task descriptions but to any input that exists and has multiple lines right?

This is correct. I will refactor.